front/public/cards/scatterplot.json

```diff
@@ -1,4 +1,5 @@
 {
+    "ribbonMessage": "Please see our newest examples below: Continuous Variables and 3D PCA.",
     "ppcalls": [
         {
             "label": "Tumor methylome tSNE",
```

Comment:
cite PNET paper

Reply:
rather than showing a CITE button, can it show `Data Source` instead?

Reply:
Yup, updated in [29dad8a](https://github.com/stjude/proteinpaint/pull/1865/commits/29dad8a3844a76f6a007c2db72b0d06993e5c27e).